robottelo/cli/sm_packages.py

```diff
@@ -51,6 +51,7 @@ def is_locked(cls, options=None):
     def status(cls, options=None):
         """Build satellite-maintain packages status"""
         cls.command_sub = 'status'
+        cls.command_end = None
```

Comment:
`command_end = None` is defined already in Base class and we're just inheriting the Base class herein Packages class, not sure what diff it makes?

Reply:
The test above fails because other commands in the sm_packages.py file like `install` and `update` set `cls.command_end = packages`. So when the command is constructed it passes the old command_end with the package name when it should not pass anything. This just makes sure we don't pass anything to the end of the status command.

Reply:
Okay, I see no issues with hardcoding `cls.command_end = None` as it's working for you.